src/do-you-want-to-stop.py

```diff
@@ -4,7 +4,7 @@
 
 answer = input('Do you want to stop? ')
 
-while answer:
-    if answer != "yes": 
-        input('Do you want to stop? ')
+while answer != "yes": 
+    input('Do you want to stop?')
+    break
```

Comment:
This unconditional break means you break every time you get here, regardless of the user's answer.